app/containers/Applications/ApplicationWizardConfirmation.tsx

```diff
@@ -212,11 +212,15 @@ export const ApplicationWizardConfirmationComponent: React.FunctionComponent<Pro
       <br />
       {revision.certificationSignatureIsValid ? (
         <>
-          <h5>
-            Thank you for reviewing the application information. The Certifier
-            has signed off on this application. You may now submit the
-            application.
-          </h5>
+          <div>
+            “By submitting the application the applicant agrees that the
```

Comment:
Much better information than my placeholder text :+1: 
Could probably use some styling though, maybe throw it in a card component with a title like 'Attention:' or 'Before you Submit:'  & put the legalese in a Card.Text, I think there's some error notifications in ApplicationWizardConfirmation that use the card element you can probably snag & replace

Reply:
Added Card